Documentation/network/clustermesh/services.rst

```diff
@@ -62,6 +62,48 @@ Below example will expose remote endpoint without sharing local endpoints.
      selector:
        name: rebel-base
 
+Synchronizing Kubernetes EndpointSlice (Beta)
+#############################################
+
+.. include:: ../../beta.rst
+
+By default Kubernetes EndpointSlice synchronization is disabled on Global services.
+If you want to be able to discover remote clusters endpoints of a Global Service
+from DNS or any third party controllers, you need to enable this behavior by adding
+the annotation ``service.cilium.io/global-sync-endpoint-slices: "true"``.
+
+Note that all the Cilium features do not need this enabled to function properly.
```

Comment:
Not clear, please rephrase. I guess you mean “not all Cilium features need this to be enabled in order to ...”, or even “Some Cilium features do not need (this annotation|synchronization) to function properly”.

Although it's also unsure how this is helpful. If I run Cilium, how do I know whether my config will be affected and whether I should turn the option on?

Reply:
Ah yes indeed this is not clear at all, this was to say that Cilium does not internally need this meaning it is only useful for third party controller/service that consume the EndpointSlice kubernetes API... So yes it need to be rephrased 😅

Reply:
I rephased this a bit, let me know!